app/components/primer/alpha/action_list.rb

```diff
@@ -25,6 +25,11 @@ class ActionList < Primer::Component
       SCHEME_OPTIONS = SCHEME_MAPPINGS.keys.freeze
 
       DEFAULT_SELECT_VARIANT = :none
+      DEFAULT_ARIA_SELECTION_VARIANT = :checked
+      ARIA_SELECTION_VARIANT_OPTIONS = [
+        :selected,
+        DEFAULT_ARIA_SELECTION_VARIANT,
+      ].freeze
       SELECT_VARIANT_OPTIONS = [
```

Comment:
```suggestion
      DEFAULT_ARIA_SELECTION_VARIANT = :checked
      ARIA_SELECTION_VARIANT_OPTIONS = [
        :selected,
        DEFAULT_ARIA_SELECTION_VARIANT,
      ].freeze
      
      DEFAULT_SELECT_VARIANT = :none
      SELECT_VARIANT_OPTIONS = [
```

A nit pick, we keep this grouped by their DEFAULT and OPTIONS